include/sdf/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ set (headers
   Filesystem.hh
   Frame.hh
   Geometry.hh
+  SatNav.hh
```

Comment:
Please move it down for alphabetical order

Reply:
https://github.com/osrf/sdformat/pull/453/commits/b128c9cc8c5aad02e8db480bd25626ae8769c545